server/chat-plugins/net-filters.ts

```diff
@@ -192,19 +221,29 @@ export const PM = new QueryProcessManager<NetQuery, any>(module, async query =>
 		}
 		return 'success';
 	case 'trainfrom':
-		const {path: targetPath, result} = data as AnyObject;
-		const content = FS(targetPath).readSync();
+		const {path, result} = data as {path: string, result: string};
+		const content = FS(path).readSync();
 		const lines = NeuralNetChecker.sanitizeChatLines(content, result);
+		const length = lines.length;
 		const time = await net.train(lines);
-		return [time, lines.length];
+		return [time, length];
 	}
-}, PM_TIMEOUT);
+}
+
+export const processes = {
+	// this one runs longer because training is SLOW
+	training: new ProcessManager.QueryProcessManager(module, handleQuery, PM_TIMEOUT),
+	// this one is fine
+	main: new ProcessManager.QueryProcessManager(module, handleQuery),
+}
 
+const PM = processes.main;
+const PMTraining = process.training;
```

Comment:
```suggestion
const PM = new ProcessManager.QueryProcessManager(module, handleQuery),
// this one runs longer because training is SLOW
const PMTraining = new ProcessManager.QueryProcessManager(module, handleQuery, PM_TIMEOUT),
```